client/src/pages/LibrarySearchPage/LibrarySearchPage.js

```diff
@@ -24,17 +199,80 @@ class LibrarySearchPage extends Component {
         <Card>
           <CardBody>
             <div className="library-search__header">
-              <Form action={artistAll}>
-                <Input className="mb-1" label="Search" name="q" icon="search" />
-                <Button type="submit">Search</Button>
-              </Form>
-              <Dropdown type="plus" text="Search">
-                <DropdownItem to="user/add">Add</DropdownItem>
-                <DropdownItem>Edit</DropdownItem>
-              </Dropdown>
+              <div>
+                <Form action={this.searchLibrary}>
+                  <Input
```

Comment:
This formatting's coming from eslint - I assume it's desired, but if it's not let me know and I'll troubleshoot es lint locally.